### PR TITLE
Add check for the prefix of an ISBN 13

### PR DESCRIPTION
**Pull request by igorframos**

### CODE REVIEW

**Comment by billdueber** [comment]
@igorframos This looks great -- my only suggestion would be to rename `#valid_prefix` to `#valid_isbn13_prefix` to make it super-clear what it's doing. I'm happy to do that myself, or you can edit and resubmit. Once I get this in (and the .travis.yml fixed) I'll release a new point version.

**Comment by michael-harrison** [comment]
+1 :)

**Comment by igorframos** [comment]
@billdueber and @michael-harrison, thanks for the review. I have updated the method name as you requested. Also, I made a small change to the implementation so it is more easily updated when new prefixes start being used in the future.

**Comment by michael-harrison** [comment]
@igorframos looks like there's bit more work to do.  There are test failures.  Looks like a stack issue with a circular call.

**Comment by igorframos** [comment]
I just want to let you know I will get to this sometime on this or on the next week. I have been a little too busy lately. =/

**Comment by igorframos** [comment]
@billdueber and @michael-harrison, could you, please, take another look?

File: lib/library_stdnums.rb [hunk 1]
```diff
@@ -184,7 +185,15 @@ def self.allNormalizedValues isbn
...
 
-
+    # Checks for a valid ISBN13 prefix
+    # ISBN13 always starts with 978 or 979. For example: 1000000000012 has a valid check digit, but
+    # is not a valid ISBN13.
+    # @param [String] isbn13 The ISBN13 to be checked.
+    # @return [Boolean] If true then the prefix is valid
+    def self.valid_isbn13_prefix?(isbn13)
+      return false unless isbn13.size == 13
+      ['978', '979'].map { |prefix| isbn13.start_with?(prefix) }.any?
```
Comment (michael-harrison) [inline lib/library_stdnums.rb:195]: You may want to consider using Regex.  It's generally faster and is much more concise.  For example:

```ruby
/^[978|979]/.match(str)
```

**Review by michael-harrison — COMMENTED** [review]
All good to go but I would strongly recommend using Regex for the prefix validation.

**Comment by billdueber** [comment]
Merging as-is, will change the prefix check to something more performant right now.

**Pull request merged**